test/integration/patients/worklist/reduced-schedule.js

```diff
@@ -252,19 +258,36 @@ context('reduced schedule page', function() {
     cy
       .routesForPatientDashboard()
       .routeCurrentClinician(fx => {
-        fx.data.id = '123456';
-        fx.data.attributes.enabled = true;
-        fx.data.relationships.role.data.id = '44444';
+        fx.data = getCurrentClinician({
+          relationships: {
+            role: getRelationship(roleReducedEmployee),
+          },
+        });
+
         return fx;
       })
       .routeActions(fx => {
-        const action = _.sample(fx.data);
+        const testAction = getAction();
+
+        const testPatientOne = getPatient({
+          attributes: {
+            first_name: 'Test',
+            last_name: 'Patient',
+          },
+        });
+
+        const testPatientTwo = getPatient({
+          attributes: {
+            first_name: 'Other',
+            last_name: 'Patient',
+          },
+        });
 
         fx.data = _.times(50, n => {
-          const clone = _.clone(action);
+          const clone = _.clone(testAction);
```

Comment:
My assumption is that this is more efficient than doing:

```js
fx.data = _.times(50, n => {
  const action = getAction();

  . . .

});
```

Doing one `getAction()` and then cloning it `50` times vs. doing a separate `getAction()` `50` times.

Reply:
Didn't see that in the schedule tests. Ya, I agree that that is cleaner 👍

Reply:
Okay I've updated these to match what's in the schedule tests.